### PR TITLE
feat: move the pin rules below the pin input

### PR DESCRIPTION
**Pull request by FarjadGov14**
# Summary of Changes

- move the pin rules below the pin input

#### Screenshot
wallet

# Related Issues

Please reference here any issue #'s that are relevant to this PR, or simply enter "N/A" if this PR does not relate to any existing issues.

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Review by MosCD3 — CHANGES_REQUESTED** [review]
Can you attach another screenshot showing inline error configured below?

**Comment by FarjadGov14** [comment]
> Can you attach another screenshot showing inline error configured below?

Attached.

**Comment by jleach** [comment]
@FarjadGov14 Would you be able to change "enought" to "enough" in the screen shot text. Just has an extra `t` in it.

**Comment by FarjadGov14** [comment]
> @FarjadGov14 Would you be able to change "enought" to "enough" in the screen shot text. Just has an extra `t` in it.

@jleach @MosCD3 Done.

**Review by MosCD3 — DISMISSED** [review]

File: packages/legacy/core/App/localization/en/index.ts [hunk 1]
```diff
@@ -217,8 +217,8 @@ const translation = {
       "NoRepetitionOfTheTwoSameNumbersValidation": "Does not contain repeating two-digit sequence.",
       "NoSeriesOfNumbersValidation": "Does not contain a series of numbers (123).",
       "PINOnlyContainDigitsValidation": "Must contain only digits.",
-      "PINTooShortValidation": "The PIN is long enought.",
-      "PINTooLongValidation": "The PIN is long enought.",
+      "PINTooShortValidation": "The PIN is long enough.",
+      "PINTooLongValidation": "The PIN is long enough.",
```
Comment (bryce-mcmath) [inline packages/legacy/core/App/localization/en/index.ts]: Should this not say "is too long."?

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]

File: packages/legacy/core/App/localization/en/index.ts (hunk 1 above)
Comment (FarjadGov14) [reply packages/legacy/core/App/localization/en/index.ts]: modified both

Comment (FarjadGov14) [reply packages/legacy/core/App/localization/en/index.ts]: @bryce-mcmath good to go

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**